core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -0,0 +1,215 @@
+package com.scalar.db.common.error;
+
+public enum CoreError implements ScalarDbError {
```

Comment:
The errors in ScalarDB Core are defined here. We will add similar definitions for other projects (GraphQL, SQL, and Cluster).